specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -4079,6 +4079,246 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataFactory/factories/{factoryName}/privateEndPointConnections": {
+      "get": {
+        "tags": [
+          "privateEndPointConnections"
+        ],
+        "operationId": "privateEndPointConnections_ListByFactory",
+        "x-ms-examples": {
+          "privateEndPointConnections_ListByFactory": {
+            "$ref": "./examples/PrivateEndPointConnections_ListByFactory.json"
+          }
+        },
+        "description": "Lists Private endpoint connections",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/factoryName"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/PrivateEndpointConnectionListResponse"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Azure Data Factory service.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataFactory/factories/{factoryName}/privateEndpointConnections/{privateEndpointConnectionName}": {
+      "put": {
+        "tags": [
+          "privateEndpointConnections"
+        ],
+        "description": "Approves or rejects a private endpoint connection",
+        "operationId": "PrivateEndpointConnection_ApprovesOrRejects",
```

Comment:
Change to PrivateEndpointConnection_CreateOrUpdate

Reply:
Done